editors/emacs/Makefile

```diff
@@ -31,6 +31,7 @@ endif
 .PHONY: install
 install: $(SRC) lambdapi-site-file.elc
 	$(INSTALL) -m 644 -t $(INSTALL_DIR) $^
+	$(MAKE) clean
```

Comment:
Could you please add a comment to say what it's for? I guess it's because of the `.install` files, but it's unusual to clean automatically after an install.

Reply:
Because make fails after a make install_emacs.

Reply:
Is it fine?

Reply:
I guess, we can't fix this now, so let's keep it and we'll have a look at it later